locales/en/filters.json

```diff
@@ -53,7 +53,7 @@
 	},
 	{
 		"filter.14.name": "AdGuard Annoyances filter",
-		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets."
+		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets filters."
```

Comment:
Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets.